src/screens/Home/components/SignUp/index.tsx

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable react/react-in-jsx-scope */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import { useForm } from 'react-hook-form';
+import i18next from 'i18next';
+
+import WoloxLogo from '../../assets/LogoWolox-Original.png';
+
+import styles from './styles.module.scss';
+
+type InputsValues = {
+  user: {
+    email: string;
+    password: string;
+    passwordConfirmation: string;
+    firstName: string;
+    lastName: string;
+    locale: string;
+  };
+};
+
+function SignUp() {
+  const { register, handleSubmit } = useForm<InputsValues>();
+  const onSubmit = (data: InputsValues) => console.log(data);
+
+  return (
+    <form className={styles.signUpForm} onSubmit={handleSubmit(onSubmit)}>
+      <img src={WoloxLogo} className={`m-bottom-3 ${styles.logo}`} />
+      <div className={`column start m-bottom-3 ${styles.inputContainer}`}>
+        <label className={`m-bottom-2 ${styles.label}`}>{i18next.t('SignUp:firstName')}</label>
```

Comment:
add to the labels a prop `htmlFor` referecing the input: https://www.w3schools.com/tags/att_for.asp

For this to work, the `input` elements should have an ìd` (you can make it match with the `name` prop). 

(I think react-hook-form already sets the `name` prop with the `register` so you don't need to add it, please check it just in case)